spec/ParseUser.spec.js

```diff
@@ -3691,7 +3692,7 @@ describe('Parse.User testing', () => {
         done();
       });
     });
-  }).pend('this test fails.  See: https://github.com/parse-community/parse-server/issues/5097');
```

Comment:
Is the combined use of `xit` and `.pend` recommended? I thought one essentially excludes the other?

Reply:
`xit.pend is not a function`

Reply:
I misread the diff file. You are removing `.pend`, not adding it. In that case we'd loose the info why this test is disabled. Wouldn't it be better to use this syntax:

```
it('should not send a verification email if the user signed up using oauth', async () => {
  pending('this test fails.  See: https://github.com/parse-community/parse-server/issues/5097');
}
```

- change from `xit` to `it`
- change from chained `.pend` to `pending()` call inside test

Reply:
I tried it your way and the test failed. I added a comment so no info is lost

Reply:
Reopened, just to investigate what `pending` didn't work. Maybe removed in jasmine 5, I'll take a look...